cmd2/cmd2.py

```diff
@@ -632,7 +632,7 @@ def perror(self, err: Union[str, Exception], traceback_war: bool=True, err_color
         err_msg = err_color + err_msg + Fore.RESET
         self.decolorized_write(sys.stderr, err_msg)
 
-        if traceback_war:
+        if traceback_war and not self.debug:
```

Comment:
Nice little fix.  It didn't make sense to print this if ``self.debug`` is ``True``.